### PR TITLE
components

### PR DESCRIPTION
**Pull request by mohamedelshamy55**
Use React components.
Use React props.
Use React state.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { Component } from 'react';
+import '../index.css';
+import Row from './Row';
+
```
Comment (rloterh) [inline src/components/Calculator.js:4]: - [ ] Your _App_ component _(App.js)_ is not classed based. Kindly change this component to a class-based component.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Mohamed, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- Descriptive Pull Request
- No linter errors

## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (mohamedelshamy55) [reply src/components/Calculator.js:4]: thank you for your comment and the problem was solved

**Review by elyor-doniyorov — APPROVED** [review]
Hi @mohamedelshamy55, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- [Optional change] Kindly, re-create the App.js, the App.js(main component) should have the imported the Calculator.js component;

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**